results/pkg/api/server/server.go

```diff
@@ -24,6 +27,11 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+const (
+	listResultsDefaultPageSize int32 = 50
+	listResultsMaximumPageSize int32 = int32((^uint32(0)) >> 1)
```

Comment:
Can this be some smaller number like 1000 or 10000? If you do intend to support page sizes up to this size it would help readability to just write out the number and add a comment describing why it's that number.